go.mod

```diff
@@ -56,119 +57,290 @@ require (
 	go.uber.org/multierr v1.6.0
 	go.uber.org/thriftrw v1.29.2
 	go.uber.org/yarpc v1.58.0
-	go.uber.org/zap v1.13.0
-	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	go.uber.org/zap v1.17.0
+	golang.org/x/net v0.0.0-20220412020605-290c469a71a5
+	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
 	golang.org/x/tools v0.1.11
 	gonum.org/v1/gonum v0.7.0
-	google.golang.org/api v0.26.0
-	google.golang.org/grpc v1.29.1
+	google.golang.org/api v0.74.0
+	google.golang.org/grpc v1.45.0
 	gopkg.in/validator.v2 v2.0.0-20180514200540-135c24b11c19
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
-	cloud.google.com/go v0.56.0 // indirect
-	cloud.google.com/go/bigquery v1.6.0 // indirect
-	cloud.google.com/go/pubsub v1.3.1 // indirect
-	github.com/BurntSushi/toml v0.3.1 // indirect
+	4d63.com/gochecknoglobals v0.1.0 // indirect
```

Comment:
(to pick a random line)
this kinda seems like a ridiculous increase in dependencies D:  are they all necessary, or could we re-do this without a `-u` / whatever pulled in so many?

either that or we should probably find out what dependency is leading to this bloat, because wow is that a lot just for doing automatic code wrapping.  +50%, though it probably has no impact on our built binaries:
```
❯ du -h vendor | tail -n1
 96M	vendor
❯ git checkout master
❯ rm -rf vendor
❯ go mod vendor
❯ du -h vendor | tail -n1
 64M	vendor
```

Reply:
tackled in https://github.com/uber/cadence/pull/4895
should be easy to adopt those changes here whenever that's done.

Reply:
It's much cleaner now!